### ISSUE TITLE
Reloading a magazine from containers that can hold a magazine

### ISSUE THREAD

**Issue by Krwak** [cataclysmbnteam/Cataclysm-BN#3117]
### Is your feature request related to a problem? Please describe.

When reloading a magazine from a clothing item that can hold a magazine and this item is already full, you can't put back the empty magazine into it but have to put it into inventory instead. - see screenshot
![image](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/72284934/9ae12f5d-9ba9-4d88-a8e9-3aabaabe2c80)
When the container isn't full in time of reloading you get instead this, which is a huge difference in movement points:
![image](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/72284934/1cde6868-5c70-49f6-a711-6d6ce7a87f42)
What this basically means that the game considers the container full, although you're only going to replace one magazine for another.
This behavior forces me to either suck up the added movement point cost or drop the magazine and retrieve it later.
Equipment during the reload when the problem occurs:
![image](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/72284934/a0543006-d665-4dfb-ac08-856e25be0eda)
This is valid for all magazine holding items that i've seen so far.

### Describe the solution you'd like

Fix the behavior of reloading the magazines into magazine holsters.
When replacing a magazine the game shouldn't consider the magazine holding item full and should allow me to insert it there as is logical.

**Comment by borsek** [comment]
Agreed, very QoL change. Can't code for shit myself, but someone might be able to figure something out. Maybe DDA has this already?
